pages/10.Components/03.Forms/02.Progressive Profiling/docs.en.md

```diff
@@ -17,7 +17,19 @@ There are two possible ways that you can configure a form field to display only
 
 It's recommended to use the email field in each form. From Mautic 2.9 email can be hidden, but be aware of that. Email as identifier of a contact could be unusable if same PC is used by more people (public library, schools...).
 
-1. Display field only if the value is not known yet
+1. Maximum fields displayed at a time
+
+A global feature to enable progressive profiling and allow the rendering of a specified amount of fields each time.
+
+![image](https://user-images.githubusercontent.com/462477/80913098-3f71c500-8d42-11ea-919c-13763228bc1c.png)
```

Comment:
```suggestion
![screenshot of maximum fields to show at a time setting](https://user-images.githubusercontent.com/462477/80913098-3f71c500-8d42-11ea-919c-13763228bc1c.png)
```

Please add this image to the repo in the folder of the page you are editing (it should be a file that you commit to Git)